.github/workflows/trigger_proverbench_dispatch.yml

```
@@ -0,0 +1,30 @@
+name: Prover Bench on halo2 PR
+on:
+  pull_request:
+    types: [labeled ,  ready_for_review]
+jobs:
+  Prover-benches-via-repo-dispatch-from-halo2-fork:
+    if: ${{ github.event.label.name == 'benchmarks' }}
+    runs-on: ubuntu-latest
+    env:
+      GH_USER: ${{ github.actor }}
+      TOKEN: ${{ secrets.PAT2 }}
+      REVISION: ${{ github.event.pull_request.head.sha }}
+      BRANCH: ${{ GITHUB_REF#refs/heads/})
```

It seems there's still a typo here
```suggestion
      BRANCH: ${{ GITHUB_REF#refs/heads/}}
```